apps/laboratory/src/components/Ethers/EthersGetCallsStatusTest.tsx

```diff
@@ -47,6 +48,10 @@ export function EthersGetCallsStatusTest() {
     }
   }
   function isGetCallsStatusSupported(): boolean {
+    // Replace with capability check
```

Comment:
Is it a todo?

Reply:
added a proper comment for following prs